bouncer/templates/base.html.jinja2

```diff
@@ -3,18 +3,22 @@
   <head>
     <meta charset=utf-8>
 
+    {% if show_metadata == True %}
```

Comment:
That fails like so:

jinja2.exceptions.TemplateAssertionError: no test named 'True'

According to https://github.com/pallets/jinja/issues/520#issuecomment-164306945:

> Jinja's is is different from Python's is. It invokes functions registered as "tests". 

namely: https://github.com/pallets/jinja/blob/master/jinja2/tests.py#L142